flask_googlemaps/templates/googlemaps/gmapjs.html

```diff
@@ -43,8 +43,8 @@
             streetViewControl: {% if gmap.streetview_control %}true{% else %}false{% endif %},
             rotateControl: {% if gmap.rotate_control %}true{% else %}false{% endif %},
             scrollwheel: {% if gmap.scroll_wheel %}true{% else %}false{% endif %},
-            fullscreenControl: {% if gmap.fullscreen_control %}true{% else %}false{% endif %}
-            styles: {{ gmap.styles | tojson  }}
+            fullscreenControl: {% if gmap.fullscreen_control %}true{% else %}false{% endif %},
```

Comment:
This is indeed the correct fix.
![image](https://user-images.githubusercontent.com/5349254/103448165-68888380-4c4a-11eb-8d06-01e371f6d5c0.png)

Reply:
I was hit by this as well, and found it out. But the bug is still there in the pip repositories.